### DISCUSSION TITLE
[Q&A] Is it possible to proxify status and inventory request from glpi server

### DISCUSSION THREAD

**Discussion by tech62** [glpi-project/glpi-agent#550] [Q&A] [answered]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

Hi,

Is it possible to "proxify" the status and inventory requests from the glpi computer details interface ?
![image](https://github.com/glpi-project/glpi-agent/assets/16237642/11f66cc5-5439-4318-87c2-880b1c4f57dc)

I want to create a frontier between my clients networks and my server network where GLPI is hosted with a "proxy" who relaying the requests.

I've seen it's possible to relay the inventory data with the glpi-agent proxy plugins, but nothing to relay these type of requests.

Thanks for your help !

**Comment by g-bougard** [comment, marked as answer]
Hi @tech62 

if I understand well, you want to know if it's possible to "proxify" the requests from GLPI to agents ?

The short to date answer is no.

In a longer answer, I can tell we started to study such kind of feature when working on [JSON protocol](https://glpi-json-protocol.readthedocs.io/en/latest/). But actually, we are still not in the process of implementing this feature, maybe in a far future release.

**Reply by tech62** [reply]
So wait & see
Thanks for your reply !
